.github/ISSUE_TEMPLATE/issue_with_content.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Lesson content issue template
+about: Let us about an issue you found with the material
+title: ''
+labels: 
+assignees: ''
+
+---
+
+<details>
+<summary><strong>Instructions</strong></summary>
+
+Thanks for taking the time to report an issue with the lesson! :heart:
+
+Please let us know about:
+
+- Technical issues or out of date screenshots and package/python versions
```

Comment:
Perhaps add material 'build' issues too?

Reply:
Particularly with build-type or similar technical issues, it's a good idea to request a concise step-by-step procedure to replicate the issue.

Reply:
Fixed via 7f1b8121272b4939ce559f578f8bfe8d8f448142.